dev-util/phpstorm/phpstorm-9.0.2.141.2462.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils versionator
+
+SLOT="$(get_major_version)"
+MY_PV="$(get_version_component_range 4-5)"
+MY_PN="PhpStorm"
+MY_PN2="webide"
```

Comment:
Now I'm being picky but please don't abuse variables too much ;-). You don't use this one anywhere else, so better just inline it in SRC_URI.